packages/dcos-diagnostics/windows.build.ps1

```diff
@@ -18,5 +18,6 @@ Copy-Item -Path "$SRC_DIR/dcos-diagnostics.exe" -Destination "$env:PKG_PATH/bin/
 
 New-Item -ItemType Directory "$env:PKG_PATH/etc"
 Copy-Item "c:/pkg/extra/dcos-diagnostics.nssm.j2" "$env:PKG_PATH/etc/"
+Copy-Item "c:/pkg/extra/servicelist.txt" "$env:PKG_PATH/bin/"
```

Comment:
Why do we need this in the `bin` directory, which normally only contains programs that can be executed? This looks like it should be in `etc`.

Reply:
I reviewed my documents from my former investigation ([see](https://github.com/DominikDary/diagnostics-win) )and there the `servicelist.txt` file had be be place close to the binary.
This my can be changed for the future, but for now I think it is fine how it is.

Reply:
the `servicelist.txt` file is used by diagnostics to contact the windows services and get their status which will be displayed in dcos ui.